package.json

```diff
@@ -2,18 +2,23 @@
   "name": "its-onboarding-exercise",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "scripts": {
     "test": "vitest --run",
     "checkFormatting": "prettier . --check",
-    "lint": "xo ."
+    "fixFormatting": "prettier . --write",
+    "lint": "xo .",
+    "build": "tsc",
+    "removeSidecar": "rimraf -g *.js && rimraf -g *.d.ts"
```

Comment:
This looks cool! I'd not heard of rimraf before, but it seems like a really useful library.